ps_spec.yml

```diff
@@ -12,18 +12,17 @@ stages:
         compute_size: small
         commands:
             - make test
-
+        
     -
         name: Master Push - Publish
         when:
             - push: ['master']
         image: node:14
         compute_size: small
         commands:
+            - make test-all
```

Comment:
let's move it under the `if` below. Let's publish coverage report only when we push new version to npm

Reply:
updated changes